ui/round/src/view/button.ts

```diff
@@ -29,12 +29,11 @@ function analysisButton(ctrl: RoundController): VNode | false {
   return (
     game.replayable(d) &&
     h(
-      'a.fbt',
+      'button.fbt',
```

Comment:
It's a `<a href>` so it can be middle-clicked into a new tab.

Reply:
yeah, i figured as much. i didn't change it for uniformity - rather the local-play behavior isn't describable via anchor link, so i split everything out in the click handler. but no reason for that now. and maybe it's better to hide this button against bots as there's no real correspondence to speak of.